http_handlers.go

```diff
@@ -209,3 +212,21 @@ func getIPFromRequest(r *http.Request) string {
 	}
 	return IPAddress
 }
+
+func handleTrustedProxies(mux *http.ServeMux) http.Handler {
```

Comment:
lets pass in the list of trusted proxies here instead of pulling it out from gxff